contrib/wofi/patches/build.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/meson.build.orig b/meson.build
+index 2697887..6d8ce4a 100644
+--- a/meson.build.orig
++++ b/meson.build
+@@ -1,4 +1,4 @@
+-project('wofi', 'c', version : 'v1.3', default_options : ['c_std=c99', 'buildtype=release', 'warning_level=2'])
```

Comment:
add a -devel subpackage too

Reply:
why? It returns "ERROR: invalid return for subpackage 'wofi-devel" tho

Reply:
> why? 

because there is a .pc file and development headers

> It returns "ERROR: invalid return for subpackage 'wofi-devel" tho

see every other devel package for how to write it